runtime/ignite/iginite.go

```diff
@@ -0,0 +1,404 @@
+package ignite
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/srl-labs/containerlab/runtime"
+	"github.com/srl-labs/containerlab/types"
+	"github.com/srl-labs/containerlab/utils"
+	api "github.com/weaveworks/ignite/pkg/apis/ignite"
+	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
+	igniteConstants "github.com/weaveworks/ignite/pkg/constants"
+	"github.com/weaveworks/ignite/pkg/dmlegacy"
+	"github.com/weaveworks/ignite/pkg/filter"
+	"github.com/weaveworks/ignite/pkg/metadata"
+	"github.com/weaveworks/ignite/pkg/operations"
+	"github.com/weaveworks/ignite/pkg/providers"
+	igniteDocker "github.com/weaveworks/ignite/pkg/providers/docker"
+	"github.com/weaveworks/ignite/pkg/providers/ignite"
+	"github.com/weaveworks/ignite/pkg/util"
+)
+
+const (
+	RuntimeName      = "ignite"
+	dockerRuntime    = "docker"
```

Comment:
can probably use the docker runtime name from `nodes` rather than redeclaring the const

Reply:
this is a default container runtime for ignite (to run its sandbox container), so it won't come from `nodes`. Unless I misunderstood you.

Reply:
yes, I meant that the `string` value `nodes.DockerRuntimeName` can be used.

Reply:
I've reworked it slightly to use constants from ignite instead. 
```go
defaultContainerRuntime       = igniteRuntimes.RuntimeDocker
defaultContainerNetworkPlugin = igniteNetwork.PluginDockerBridge
```
This way it's consistent for both runtime and network plugin and also indicates where this dependency is coming from. Hope that's ok?